examples/record_mac.py

```diff
@@ -1,14 +1,18 @@
+from __future__ import print_function
```

Comment:
no need for that, just use the brackets correctly :)

Reply:
For me it's ok to remove.
What about print('i got', len(data)) in python2 ? Without this import returns a tuple.

Reply:
You're right. Removed.